src/Orchard.Tests.Modules/Migrations/SchemaCommandGeneratorTests.cs

```diff
@@ -118,6 +118,7 @@ public class SchemaCommandGeneratorTests {
         }
 
         public class StubFolders : IExtensionFolders {
+            public IEnumerable<string> Paths { get { return AvailableExtensions().Select(e => e.Location).Distinct(); } }

```

Comment:
Indeed. However the class itself is repeated several times, so if touching this It may make sense to reduce that redundancy on the first place.. 
Since all the repetitions are part of various tests I would leave this to after we verify all else works..